frontend/src/services/handlePlantApi.js

```diff
@@ -0,0 +1,89 @@
+const baseUrl = 'http://urbanplants.local/plant'
+
+export async function getPlants() {
+  try {
+    const response = await fetch(baseUrl)
+    const data = await response.json()
+    return await data
+  } catch (error) {
+    return { error: 'The server is down! :(' }
+  }
```

Comment:
You are loosing information like the error stack here. This makes it difficult to investigate issues, because you don't have a useful error message anymore.

Reply:
As I know so far I only catch if the server is not responding, what other type of error does it catch that I should display?

Reply:
There are many possible errors like "timeout", "JSON.parse failed" (if the server is not sending JSON), "Invalid request" due to changes in your components, "CORS", and many others.

But you override the error message with `The server is down! :(` in any cases.
If you catch an error, I would expect to handle it (like showing a message to your users or try again). 
Thus, I would remove all try-catch in your services and catch possible errors in your Components.
The error message and stack is useful to identify which issue happened.

Reply:
Thanks, I'm going to change the whole error catching and handling in a later story, thanks for the input 👍 😃